conda/recipes/ucxx/recipe.yaml

```diff
@@ -0,0 +1,349 @@
+            - include/ucxx/utils/python.h
+            - include/ucxx/utils/sockaddr.h
+            - include/ucxx/utils/ucx.h
+            - include/ucxx/worker.h
+            - include/ucxx/worker_progress_thread.h
+            - lib/libucxx.so
+    about:
+      homepage: ${{ load_from_file("python/libucxx/pyproject.toml").project.urls.Homepage }}
+      license: ${{ load_from_file("python/libucxx/pyproject.toml").project.license.text }}
+      summary: ${{ load_from_file("python/libucxx/pyproject.toml").project.description }}
+
+  - package:
+      name: libucxx-examples
+      version: ${{ version }}
+    build:
+      string: ${{ date_string }}_${{ head_rev }}
+      script: cmake --install cpp/build --component examples
+      dynamic_linking:
+        overlinking_behavior: "error"
+    requirements:
+      build:
+        - cmake ${{ cmake_version }}
+        - ${{ stdlib("c") }}
+      host:
+        - libucxx
+      run:
+        - ${{ pin_subpackage("libucxx", exact=True) }}
+      ignore_run_exports:
+        by_name:
+          - cuda-cudart
+          - cuda-version
+          - librmm
+          - libucxx
+          - python_abi
+          - if: cuda_major == "11"
+            then: cudatoolkit
+    about:
+      homepage: ${{ load_from_file("python/libucxx/pyproject.toml").project.urls.Homepage }}
+      license: ${{ load_from_file("python/libucxx/pyproject.toml").project.license.text }}
+      summary: libucxx examples executables
+
+  - package:
+      name: libucxx-tests
+      version: ${{ version }}
+    build:
+      string: cuda${{ cuda_major }}_${{ date_string }}_${{ head_rev }}
+      script: cmake --install cpp/build --component testing
+      dynamic_linking:
+        overlinking_behavior: "error"
+    requirements:
+      build:
+        - cmake ${{ cmake_version }}
+        - ${{ stdlib("c") }}
+      host:
+        - cuda-version =${{ cuda_version }}
+        - if: cuda_major == "11"
+          then: cudatoolkit
+          else: cuda-cudart-dev
+        - libucxx
+      run:
+        - if: cuda_major == "11"
+          then: cudatoolkit
+        - ${{ pin_compatible("cuda-version", upper_bound="x", lower_bound="x") }}
+        - ${{ pin_subpackage("libucxx", exact=True) }}
+      ignore_run_exports:
+        by_name:
+          - cuda-cudart
+          - cuda-version
+          - librmm
+          - libucxx
+          - python_abi
+          - ucx
+          - if: cuda_major == "11"
+            then: cudatoolkit
+    about:
+      homepage: ${{ load_from_file("python/libucxx/pyproject.toml").project.urls.Homepage }}
+      license: ${{ load_from_file("python/libucxx/pyproject.toml").project.license.text }}
+      summary: libucxx tests executables
+
+  - package:
+      name: ucxx
+      version: ${{ version }}
+    build:
+      string: cuda${{ cuda_major }}_py${{ py_version }}_${{ date_string }}_${{ head_rev }}
+      # dynamic_linking:
+      #   overlinking_behavior: "error"
```

Comment:
Can we remove this?

Reply:
That's a TODO before this gets merged -- `libucxx_python.so` and others link against `libpython.so` but this is flagging the overlinking checks in `rattler` for reasons I haven't figured out yet.

Reply:
Ok, I'll switch my review to request change just to remind ourselves of that.